### ISSUE TITLE
Empty literals removed

### ISSUE THREAD

**Issue by ate47** [sparqling/sparql-formatter#14]
Hello, I was using the online formatter (https://sparqling.github.io/sparql-formatter/) and I noticed that the empty literals are removed, even with a type/language

```SPARQL
SELECT * { ?s ?p [ ?p2 """hello 
             ddqzdzq    dzq   
            qz"""^^<http://example.com/#test>; <http://example.com/#p> "" ] . ?qzd ?zdq       ""^^<http://example.com/#test2> .
         [] ?ddzdq       ""@zdq}
```
is converted to
```SPARQL
SELECT *
WHERE {
  ?s ?p [ ?p2 """hello 
             ddqzdzq    dzq   
            qz"""^^<http://example.com/#test> ;
       <http://example.com/#p>  ] .
  ?qzd ?zdq  .
  [] ?ddzdq  .
}
```

Notice that the `""` are removed

**Comment by hchiba1** [comment]
Thanks for the report!
This issue is fixed in the latest version (v0.6.3, https://sparqling.github.io/sparql-formatter/).
Now, the output will be:
```SPARQL
SELECT *
WHERE {
  ?s ?p [ ?p2 """hello 
             ddqzdzq    dzq   
            qz"""^^<http://example.com/#test> ;
       <http://example.com/#p> "" ] .
  ?qzd ?zdq ""^^<http://example.com/#test2> .
  [] ?ddzdq ""@zdq .
}
```

**Comment by ate47** [comment]
Thanks!
